types/module/core_module.go

```diff
@@ -112,15 +112,18 @@ func (c coreAppModuleAdaptor) ExportGenesis(ctx context.Context) (json.RawMessag
 	}
 
 	if mod, ok := c.module.(HasABCIGenesis); ok {
-		return mod.ExportGenesis(ctx)
+		eg, err := mod.ExportGenesis(ctx)
```

Comment:
not needed, we can just return as it was

Reply:
your snippet does the same thing as previously but is just more verbose, this is why it can be reverted.

Reply:
o, i see, i will do it